config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
-  root 'home#index'
+  resources :home
+  get "show"=>"home#show"
+  root to: 'home#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.